frontend/lib/rental-history.tsx

```diff
@@ -22,10 +22,11 @@ import { ClearSessionButton } from './clear-session-button';
 import { OutboundLink } from './google-analytics';
 import { CustomerSupportLink } from './customer-support-link';
 import { updateAddressFromBrowserStorage } from './browser-storage';
+import { GetStartedButton } from './get-started-button';
 
 const RH_ICON = "frontend/img/ddo/rent.svg";
 
-function RentalHistoryWelcome(): JSX.Element {
+function RentalHistorySplash(): JSX.Element {
```

Comment:
I renamed this function because it was actually misleading... It's not a great nomenclature, but "splash" is for pages that don't require login and exist at `/splash` (which this one does), while "welcome" is for pages that require login and are at `/welcome`.